vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -61,7 +61,7 @@ $vsComponents = @('Microsoft.Component.MSBuild',
     'Microsoft.VisualStudio.Component.VC.Tools.x86.x64',
     'Microsoft.VisualStudio.ComponentGroup.UWP.Support',
     'Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core',
-    'Microsoft.VisualStudio.Component.Windows10SDK.19041');
+    'Microsoft.VisualStudio.Component.Windows10SDK.22000');
```

Comment:
This is failing PR validation.

Do we need to apply an update to the build agent image?